nucleus-core/src/main/java/io/github/nucleuspowered/nucleus/modules/nickname/services/NicknameService.java

```diff
@@ -174,7 +181,7 @@ public void removeFromCache(UUID player) {
 
     @Override
     public Optional<Text> getNickname(UUID user) {
```

Comment:
I wonder if a better option would be to alter/overload this to accept a boolean force parameter to bypass the cache check. I might also redo the cache to support offline players and have a time based expiry on the data using Caffeine.

This will work for now though, thanks!